README.md

```diff
@@ -37,12 +37,10 @@ source .venv/bin/activate
 
 ## Usage
 
-The following example demonstrates a full ingestion workflow using Spark, MinIO, and LinkML schema integration.
+The following example demonstrates a full ingestion workflow using Spark and MinIO.
 
 ```python
-from data_lakehouse_ingest import data_lakehouse_ingest_config
 from data_lakehouse_ingest.logger import setup_logger
```

Comment:
what elements of the logger does the user have to set up? It seems as though this is something that would be better off for the dlh_ingest framework to create, using variables supplied in the config file.